modules/installation-aws-delete-cluster.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-delete-cluster"]
+= Deleting a cluster with a configured AWS Local Zone infrastructure
+
+After you install a cluster on Amazon Web Services (AWS) into an existing Virtual Private Cloud (VPC), and you set subnets for each Local Zone location, you can delete the cluster and any AWS resources associated with it. 
+
+The example in the procedure assumes that you created a VPC and its subnets by using a CloudFormation template.
+
+.Prerequisites
+
+* You know the name of the CloudFormation stack, such as `cluster-vpc`. You need the name of the stack to delete the cluster.
+* You can access rights to the directory that contains the installation program. This directory stores any files created by the installation program. 
```

Comment:
-  contains the installation program
- created by the installation program

those sentences become confusing, IMO. The directory contains the files created by the installation program, not necessarily the program need to be there. If the program is there (same dir of install files) you may not need to specify the --dir. I think the first part of this phase must be reviewed.

Reply:
Good point, @mtulio . I updated to state:

> You can access rights to the directory that contains the installation files that were created by the installation program.